onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -305,7 +305,9 @@ def torch_graph(self):
         return self._torch_graph
 
     @beartype
-    def add_input(self, input_name: str, input_value: torch.Tensor) -> TorchScriptTensor:
+    def add_input(
+        self, input_name: str, input_value: Optional[torch.Tensor] = None
```

Comment:
We prob always want users to specify the input value. So I would remove the ` = None` part.

Reply:
I will put this in https://github.com/microsoft/onnx-script/pull/416